src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a,
+p {
+  padding: 0;
+  margin: 0;
+}
+
+.card {
+  background: #fff;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__title {
+    margin-top: 36px;
+    margin-left: 16px;
+    width: 166px;
+    height: 36px;
+  }
+
+  &__product-name {
+    color: #060b35;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product {
+    margin-top: 4px;
+  }
+
+  &__product-code {
+    color: #616070;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 18px;
+    padding-left: 2px;
+  }
+
+  &__review-text {
+    color: #060b35;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 22px;
+  }
+
+  &__price-text {
+    color: #616070;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    color: #060b35;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buttom {
+    background-color: #00acdc;
+    width: 166px;
+    height: 40px;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    text-align: center;
+    line-height: 38px;
+    margin-top: 16px;
+
+    a {
+      text-decoration: none;
+      font-weight: 700;
+      font-size: 14px;
+      color: #fff;
+    }
+
+    &:hover {
+      background-color: #fff;
+
+      a {
+        color: #00acdc;
+      }
+    }
+  }
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
```

Comment:
Please, change this implementation to a modifier as in Stars task, as requested in the checklist